hook!-full-game/src/Levels/Exit.gd

```diff
@@ -0,0 +1,25 @@
+tool
+extends Area2D
+
+
+export(String, FILE) var next_level_path: String = ""
```

Comment:
Have you tried using a PackedScene directly? Using PackedScene we could drag and drop the levels into the inspector's slot. What do you think is best?

Not sure which is best. Also `next_level_path: String = ""` -> `next_level_path: = ""`, no need for the type hint

Reply:
Doesn't work with PackedScene directly if we want to do cyclic paths: `Level1`'s `Exit` points to `Level2` & `Level2`'s `Exit` points to `Level1` wouldn't work, that's why I ended up  with `export(String, FILE)`

Reply:
Got it!